travelpal/assets/js/components/LoginForm.jsx

```diff
@@ -34,7 +34,7 @@ function LoginForm(props) {
     <React.Fragment>
       <div className="header">
         {/* Image source: https://www.shareicon.net/transport-flight-aeroplane-airplane-airport-transportation-travel-plane-824400 */}
-        <img src="images/airplane-icon.png" alt="logo" />
+        <img src="/images/airplane-icon.png" alt="logo" />
```

Comment:
Need to make this change in `Nav.jsx` as well. After logging out and then logging in, the logo didn't load for me until after I added the `/` in `Nav.jsx`.

Not sure why the logo shows up the first time I log in before making this change, though.

Reply:
I takes part of the current url and adds it to the img path, thats why

Reply:
https://github.com/kimberlypn/TravelPal/pull/20/commits/700aa97d0478caa7ee087f41897391662adcfd3f